contracts/senders/aave-v2-migrator/variables.sol

```diff
@@ -29,28 +30,28 @@ contract Variables {
         address[] borrowTokens;
     }
 
-    struct TokenPrice {
-        uint priceInEth;
-        uint priceInUsd;
-    }
+    // Constant Addresses //
 
     /**
-     * @dev Aave referal code
-     */
+    * @dev Aave referal code
+    */
     uint16 constant internal referralCode = 3228;
-
-    address constant internal polygonReceiver = 0xA35f3FEFEcb5160327d1B6A210b60D1e1d7968e3; // This is fake addr. TODO: Replace this
-    FlashloanInterface constant internal flashloanContract = FlashloanInterface(0x709407B70ce0845996108205aA92aE87B22F7b1a); // TODO: Replace this
+    
+    /**
+    * @dev Polygon Receiver contract
+    */
+    address constant internal polygonReceiver = 0xA35f3FEFEcb5160327d1B6A210b60D1e1d7968e3; // This is fake addr. TODO @thrilok209: Replace this
+    
+    /**
+    * @dev Flashloan contract
+    */
+    FlashloanInterface constant internal flashloanContract = FlashloanInterface(0x709407B70ce0845996108205aA92aE87B22F7b1a); // TODO @thrilok209: Replace this
```

Comment:
We have to replace these addresses. Right?

Reply:
Yes. I will replace before deployment.